app/javascript/admin/__tests__/controllers/article_controller.test.js

```diff
@@ -1,5 +1,5 @@
 import { Application } from 'stimulus';
-import { ArticleController } from '../../controllers/article_controller';
+import ArticleController from '../../controllers/article_controller';
```

Comment:
All Stimulus controllers need to be default exports. The new eslint rule we added did not take this into account and we missed this when smoke testing #12512.